samples/matter/common/src/events/matter_event_handler.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include <platform/PlatformManager.h>
+
+namespace Nordic
+{
+namespace Matter
+{
+	/**
+	 * @brief Safely register custom Matter event handler
+	 *
+	 * This function adds user provided Matter event handler in a thread safe manner.
```

Comment:
Adding custom handler will not automatically unregister `DefaultEventHandler`, right? Maybe it would be good to add one sentence that it extends the `DefaultEventHandler` functionality and it is not going to replace it (first I thought it is going to override it).

Reply:
Yes, we can register multiple handlers (these are kept as a linked list in the server). I will add a comment on that.